lib/al/Library/Layout/LayoutActorUtil.h

```diff
@@ -1,19 +1,116 @@
 #pragma once
 
+#include <math/seadMatrix.h>
 #include <math/seadVector.h>
 
-namespace al {
-class LayoutActor;
-class IUseLayout;
+namespace nn::ui2d {
+class TextureInfo;
+}
 
-bool isActive(al::LayoutActor* actor);
-bool isDead(al::LayoutActor* actor);
+namespace sead {
+class Color4u8;
+}
 
-void killLayoutIfActive(al::LayoutActor* actor);
-void appearLayoutIfDead(al::LayoutActor* actor);
+namespace agl {
+class TextureData;
+}
 
-void calcTrans(sead::Vector3f* trans, const al::IUseLayout*);
-const sead::Vector3f& getLocalTrans(const al::IUseLayout*);
+namespace al {
+class LayoutActor;
+class LayoutPaneGroup;
+class IUseLayout;
+class MessageHolder;
 
-void updateLayoutPaneRecursive(al::LayoutActor* actor);
+bool killLayoutIfActive(al::LayoutActor*);
+bool appearLayoutIfDead(al::LayoutActor*);
+bool isActive(const al::LayoutActor*);
+bool isDead(const al::LayoutActor*);
+void calcTrans(sead::Vector3f*, const al::IUseLayout*);
+sead::Vector3f& getLocalTrans(const al::IUseLayout*);
+sead::Vector3f* getLocalTransPtr(const al::IUseLayout*);
+void calcScale(sead::Vector3f*, const al::IUseLayout*);
+sead::Vector3f& getLocalScale(const al::IUseLayout*);
```

Comment:
```suggestion
const sead::Vector3f& getLocalScale(const al::IUseLayout*);
```
same below on a few others